### PR TITLE
Fix linter errors

### PR DESCRIPTION
**Pull request by tiagojsag**

### CODE REVIEW

**Comment by tiagojsag** [comment]
Down to 34 ;)

File: app/src/components/map.jsx [hunk 1]
```diff
@@ -211,28 +217,6 @@ class Map extends Component {
   }
 
   /**
-   * Draws a line that connects by timestamp order all paths of the following vessel
```
Comment (nerik) [inline app/src/components/map.jsx]: Are you sure we want to remove that bit?
Before that I was able to click on vessels and see the red path.

Comment (tiagojsag) [inline app/src/components/map.jsx]: AFAIK this is dead code, it was replaced when we added the vessel tracks using the detailed data

Comment (nerik) [inline app/src/components/map.jsx]: Just sayin', I've tested it and it seems *something* is broken.

**Pull request merged**